euth/actions/emails.py

```diff
@@ -0,0 +1,20 @@
+from euth.contrib import emails
+
+
+def notify_creator_on_create_action(action):
```

Comment:
The thing here is, I am trying to deprecate the Email sending via the `send_email_with_template` function and replace it with the class-based emails (comparable to class-based views). Currently there are two places left: `reports`  and `users`. Reports can be changes as soon as the report dashboard tab is ready.